analyses/cnv-chrom-plot/util/bin-coverage.R

```diff
@@ -139,11 +139,11 @@ call_bin_status <- function(sample_id,
     # the frac_delta_threshold
     dplyr::mutate(
       status =  dplyr::case_when(
-        frac_uncallable > uncallable_threshold ~ "uncallable"
-        frac_gain > threshold ~ "gain",
-        frac_loss > threshold ~ "loss",
-        frac_neutral > threshold ~ "neutral",
-        TRUE ~ "unstable"
+        frac_uncallable > frac_uncallable_val ~ "uncallable",
```

Comment:
Is this what we wanted here? Or should we keep what's in master?

Reply:
It looks like this is correct, as it now matches the function arguments. However, the docs for the argument should also be changed to match.

Reply:
Unless you were looking at the order of the final two options, but I _think_ that what you have is correct, because you got strange results the other way? I don't fully remember.

Reply:
Yes. I think we had settled on this. https://alexslemonade.slack.com/archives/CNH4FND1C/p1586786889017900

Will make sure docs are updated though.